src/Framework/TestCase.php

```diff
@@ -1948,6 +1948,14 @@ public function dataDescription()
         return \is_string($this->dataName) ? $this->dataName : '';
     }
 
+    /**
+     * @return int|string
+     */
+    public function dataID()
```

Comment:
I am not convinced (yet) that exposing this information is required. In any case, the method should be named `dataName()`.

Reply:
I had the impression that "name" would be string only, so I used ID for int|string. I'm ok with any name though.